### DISCUSSION TITLE
Customize minWidth of cards

### DISCUSSION THREAD

**Discussion by MeriemKhadim** [prabhuignoto/react-chrono#430] [Q&A]
Please I need to customize the minWidth of the card. I tried cardWidth but it seems like it sets the maximum width not the minimum.

**Comment by prabhuignoto** [comment]
@MeriemKhadim have you tried using custom class names https://github.com/prabhuignoto/react-chrono#using-custom-class-names
you can target specific elements in the timeline through this and apply your own custom styles.
